packages/pwa/tests/lighthouserc.js

```diff
@@ -21,11 +26,11 @@ module.exports = {
         assert: {
             aggregationMethod: 'median',
             assertions: {
-                //TODO: Adjust scores and the assertions level from 'warn' to 'error' to fail the build if the scores are not met.
-                'categories:performance': ['warn', {minScore: 0.8}],
-                'categories:pwa': ['warn', {minScore: 0.9}],
-                'categories:seo': ['warn', {minScore: 0.9}],
-                'categories:accessibility': ['error', {minScore: 0.96}]
+                // NOTE: Adjust the scores accordingly as the performance is improved
+                'categories:performance': ['error', {minScore: 0.3}],
+                'categories:pwa': ['error', {minScore: 0.9}],
+                'categories:seo': ['error', {minScore: 0.85}],
+                'categories:accessibility': ['error', {minScore: 0.88}]
```

Comment:
Open question: but how will this effect generated projects? Should be be enforcing lighthouse tests on generated projects? What happens if they delete a page, will they know to delete the entry in the urls to test? This has a similar tone to the change Oli did for the eslint change [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/104)..

thoughts?

Reply:
Good question. I don't think we should enforce lighthouse tests on  a generated project. Since there is not CI on it, this file simply acts as a recommendation for the lighthouse tests. If the customer wants to use it, they will probably go look at this file and adjust it accordingly to their needs, if not, they can choose to use different approach and delete this file.